codegenerator/cli/templates/dynamic/codegen/src/ContextEnv.res.hbs

```diff
@@ -119,11 +119,9 @@ let makeEntityHandlerContext = (
   ~logger,
   ~getKey,
   ~loadLayer,
-  ~isInReorgThreshold,
+  ~shouldSaveHistory,
 ): entityHandlerContext<entity> => {
   let inMemTable = inMemoryStore->InMemoryStore.getInMemTable(~entityMod)
-  let shouldSaveHistory =
-    RegisterHandlers.getConfig()->Config.shouldSaveHistory(~isInReorgThreshold)
```

Comment:
Another place to remove the config lookup at the file level to ensure tests behave accordingly. "shouldSaveHistory" is now calculated and handed down.